pkg/container-utils/docker/docker.go

```diff
@@ -81,7 +81,18 @@ func listContainers(c *DockerClient, filter *dockerfilters.Args) ([]dockertypes.
 			opts, err)
 	}
 
-	return containers, nil
+	noPauseContainers := []dockertypes.Container{}
+	for _, c := range containers {
+		if c.Labels["io.kubernetes.docker.type"] == "podsandbox" {
+			// Temporarily ignoring pod sandbox containers. Otherwise, they will
+			// be considered as containers and EnrichByNetNs will incorrectly
+			// think that they are using a given network namespace. See issue
+			// https://github.com/inspektor-gadget/inspektor-gadget/issues/1095.
+			continue
+		}
+		noPauseContainers = append(noPauseContainers, c)
+	}
+	return noPauseContainers, nil
```

Comment:
The proper solution for this should be:

- Always include pause containers: Do not ignore them here in Docker, and include them with Containard and CRI-O as described in https://github.com/inspektor-gadget/inspektor-gadget/issues/1095. 
- Add a boolean in the `Container` struct that says whether the container is a sandbox container.
- `EnrichByNetNs` should consider whether the container is a sandbox or not to decide whether to include the container information.